src/project.cc

```diff
@@ -620,7 +629,8 @@ std::vector<Project::Entry> LoadCompilationEntriesFromDirectory(
 
   if (!g_config->compilationDatabaseCommand.empty()) {
 #if defined(_WIN32)
-// TODO
+    _unlink((comp_db_dir + "compile_commands.json").c_str());
```

Comment:
Ideally this should go in platform.h as well, but if not that's okay.

Reply:
would you prefer to have two functions `unlink()` and `rmdir()` or a single function `rmdirRecursive()`?

Reply:
Probably `RemoveDirectoryRecursive` next to `MakeDirectoryRecursive`